packages/fleet/lib/models/nodes.integration.test.ts

```diff
@@ -125,37 +132,22 @@ describe('Nodes', () => {
         expect(searchThirdPage.nodes.size).toBe(2);
         expect(searchThirdPage.nextCursor).toBe(undefined);
     });
-});
-
-async function createNodeWithState(
-    db: knex.Knex,
-    { state, deploymentId, routingId = nanoid() }: { state: NodeState; deploymentId: number; routingId?: RoutingId }
-): Promise<Node> {
-    let node = await createNode(db, { routingId, deploymentId });
-    if (state == 'ERROR') {
-        return (await nodes.fail(db, { nodeId: node.id, error: 'my error' })).unwrap();
-    }
-    // transition to the desired state
-    while (node.state !== state) {
-        const nextState = nodes.validNodeStateTransitions.find((v) => v.from === node.state)?.to;
-        if (nextState) {
-            node = (await nodes.transitionTo(db, { nodeId: node.id, newState: nextState })).unwrap();
-        } else {
-            throw new Error(`Cannot transition node to state '${state}'`);
-        }
-    }
-    return node;
-}
-
-async function createNode(db: knex.Knex, { routingId, deploymentId }: { routingId: RoutingId; deploymentId: number }): Promise<Node> {
-    const node = await nodes.create(db, {
-        routingId,
-        deploymentId,
-        url: 'http://localhost:1234',
-        image: 'nangohq/my-image:latest',
-        cpuMilli: 500,
-        memoryMb: 1024,
-        storageMb: 512
+    it('should be able to fail a node', async () => {
+        const node = await createNodeWithAttributes(db, { state: 'PENDING', deploymentId: activeDeployment.id });
+        const failedNode = (await nodes.fail(db, { nodeId: node.id, reason: 'my error' })).unwrap();
+        expect(failedNode.state).toBe('ERROR');
+        expect(failedNode.error).toBe('my error');
+    });
+    it('should be able to register a node', async () => {
```

Comment:
I'm not sure what you have against line break 🤣

Reply:
do you mean empty line between the tests? :)

Reply:
yes even in general, between logical code blocks but I guess it's keyboard-driven development :p